doc/command_ref.rst

```diff
@@ -1515,6 +1515,11 @@ Shell Command
         * reset: reset the transaction
         * run: resolve and run the transaction
 
+    Note that all local packages must be used in the first shell transaction subcommand like
+    `install /tmp/nodejs-1-1.x86_64.rpm /tmp/acpi-1-1.noarch.rpm` otherwise an error appears. Also
+    `disable`, `enable`, and `reset` module operations (like `module enable nodejs`) must be
+    performed before any other shell transaction subcommand is used.
```

Comment:
> Note that all local packages must be used in the first shell transaction subcommand **(e.g.**
`install /tmp/nodejs-1-1.x86_64.rpm /tmp/acpi-1-1.noarch.rpm`**)** otherwise an error **will occur**. **Any**
**`enable`, `disable`**, **or** `reset` module operations (**e.g.** `module enable nodejs`) must **also** be
performed before any other shell transaction subcommand is used.